tests/installation/scc_registration.pm

```diff
@@ -20,6 +20,16 @@ use testapi;
 use registration;
 
 sub run() {
+    if (get_var("SCC_EXPECT_ERROR")) {
+        while (1) {
+            if (check_screen 'registration-error') {
+                send_key 'alt-o';
+                sleep 2;
```

Comment:
sleep 2 has purpose, to be human ... otherwise openQA will press alt-o three times, twice until pop-up disappear and another alt-o will press "local registration server"  http://10.100.98.90/tests/1782/modules/scc_registration/steps/4

I like sleep and don't like to replace sleep 2 with something else which will do the same but with unnecessary screen asserting and use lot of code `eq` (killing fly with 1k nuke)

e.g. wait_still_screen is pretty complex API

I will change it if you really want it

```perl
sub wait_still_screen {
    my $stilltime        = shift || 7;
    my $timeout          = shift || 30;
    my $similarity_level = shift || (get_var('HW') ? 44 : 47);

    bmwqemu::log_call('wait_still_screen', stilltime => $stilltime, timeout => $timeout, simlvl => $similarity_level);

    $timeout = bmwqemu::scale_timeout($timeout);

    my $starttime      = time;
    my $lastchangetime = [gettimeofday];
    $bmwqemu::backend->set_reference_screenshot;

    while (time - $starttime < $timeout) {
        my $sim = $bmwqemu::backend->similiarity_to_reference->{sim};
        my $now = [gettimeofday];
        if ($sim < $similarity_level) {

            # a change
            $lastchangetime = $now;
            $bmwqemu::backend->set_reference_screenshot;
        }
        if (($now->[0] - $lastchangetime->[0]) + ($now->[1] - $lastchangetime->[1]) / 1000000. >= $stilltime) {
            bmwqemu::fctres('wait_still_screen', "detected same image for $stilltime seconds");
            return 1;
        }
        sleep(0.5);
    }
    $autotest::current_test->timeout_screenshot();
    bmwqemu::fctres('wait_still_screen', "wait_still_screen timed out after $timeout");
    return 0;
}
```